client/gutenberg/editor/main.jsx

```diff
@@ -32,14 +31,26 @@ class GutenbergEditor extends Component {
 	}
 
 	render() {
-		if ( isEmpty( this.props.siteSlug ) ) {
+		const { siteId, siteSlug, post } = this.props;
+
+		if ( isEmpty( siteSlug ) ) {
 			return null;
 		}
 
-		applyAPIMiddlewares( this.props.siteSlug );
+		applyAPIMiddlewares( siteSlug );
```

Comment:
noting here again - this probably belongs in `componentDidMount()` - see #27009

Reply:
I'll create a janitorial PR with that change to keep this PR focused on how to get the data from the server and where to put it 😉 .

Reply:
didn't mean to imply you needed any change here, but more an alert to be aware that this line is going to change on you